docs/diagrams/business_models.puml

```diff
@@ -0,0 +1,57 @@
+@startuml BusinessModels
+
+object event
+object day
+object room
+object timeslotSize
+object timeslot
+object session
+object style
+
+event : name = "Name of the Event/Conference"
+event : description = "Description of the Event/Conference"
+event : startDate = "Start Date of the Event/Conference"
+event : endDate = "End Date of the Event/Conference"
+event : style = style
+event : days = day[]
+event : timeslotSizes = timeslotSizes[]
+
+day : date = "Date of the Day"
+day : note = "A note for special days"
+day : rooms = room[]
+
+room : name = "Name of the Room"
+room : timeslots = timeslot[]
+
+timeslotSize : size = integer
```

Comment:
brauchen wir die timeslotSize?

Reply:
Ja. Du legst z.B. 7 Timeslots an:
1. mittagessen = 60min
2. Kaffepause = 10min
3. lightnintalk = 15min
4. Kleine Session = 30min
5. Normalle Session = 45min
6. Keynote = 60min
7. Workshop = 420min

Da brauchst du zum zuweisen der timeslotsize (typen) zu den timeslots der Räume ja eine größe in minuten.

Reply:
Wieso kann ich den timeslots nicht einfach direkt die länge in minuten zuweisen?

Reply:
Das kann man auch. Das ist jetzt nur meine Idee, dass es einfacher sein kann, wenn man die unterschiedlichen Größen der Slots im Prozess vordefiniert und dann sie entsprechend auf die Räume verteilen. 
Wenn man die Größe bei jeden Slot definiert, dann muss man aufpassen, dass die Längen immer gleich sind bzw. wahrscheinlich wird die Person dann mit einem „Excel Block“ daneben um die unterschiedlichen Slots Längen im Blick zu halten.

Aber wie gesagt ist meine Idee, weil ich mir denke, dass es so die Arbeit des Planers von dem Event vereinfachen könnte.